cocoa/src/toga_cocoa/app.py

```diff
@@ -297,6 +303,17 @@ def _create_app_commands(self):
                 group=toga.Group.HELP,
             ),
         )
+        if macos_version_at_least(10, 12):
+            self.interface.commands.add(
+                toga.Command(
+                    self._menu_merge_all_windows,
+                    "Merge All Windows",
+                    group=toga.Group.WINDOW,
```

Comment:
This results in a "merge all windows" menu item that comes before "minimize". From what I can see, Minimise should be in a section, with "merge all" in a separate, higher-valued section.

At the risk of scope creep, I'd be interested in seeing how many of the other standard macOS window commands could be implemented.

Reply:
Ah, good point. I need to take more of a look at how Toga organizes commands.